src/test/java/com/cloudant/tests/ReplicateBaseTest.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Your `52593-replicate-tests` branch did help - all tests pass on travis that I had originally ignored.